src/usr/local/www/classes/Form/Checkbox.class.php

```diff
@@ -56,9 +56,9 @@ protected function _getInput()
 	{
 		$input = parent::_getInput();
 
-		if (empty($this->_description))
-			return '<label class="chkboxlbl">'. $input .'</label>';
+		if (!empty($this->_description) || is_numeric($this->_description))
```

Comment:
I guess you were probably kidding but anyway, no, we are not patching pfSense PHP to behave different from the original. PHP is what it is :)